### DISCUSSION TITLE
Hope to have a better farming pool support

### DISCUSSION THREAD

**Discussion by zjshen14** [Chia-Network/chia-blockchain#3481] [Ideas]
Small players deserves some hope. Please support https://github.com/Chia-Network/chia-blockchain/issues/3479.

**Comment by unitedchia** [comment]
There are already a lot of pools announced for the official pooling protocol. You can find them listed on [poolminingstats.stream](https://miningpoolstats.stream/chia).

One of the is us [chiaunited.com](chiaunited.com). We are trying to provide a low fee and reliable pool for chia farmers.
Feel free to visit and subscribe to any news about the pool launch.
